### PR TITLE
Implemented interactive microphone

### PR DESCRIPTION
**Pull request by m-voloshin**

### CODE REVIEW

File: modules/UI/videolayout/VideoLayout.js [hunk 1]
```diff
@@ -123,14 +123,32 @@ var VideoLayout = {
...
+            this.getVideoThumbnailById(id).imic.volume(lvl * 100).redraw();
+            return;
+        }
+
         AudioLevels.updateAudioLevel(
             id, lvl, largeVideo.id
         );
     },
 
+    getVideoThumbnailById(id) {
```
Comment (yanas) [inline modules/UI/videolayout/VideoLayout.js]: JSDocs missing. Also if we introduce this function we should make sure to use wherever possible to get a video object.

File: modules/UI/videolayout/InteractiveMicrophone.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+            elem.find('.mic-level').css({
+                display: micDisplayStyle,
+                height: level + '%'
+            });
+        });
+
+        return this;
+    }
+
+    render () {
```
Comment (yanas) [inline modules/UI/videolayout/InteractiveMicrophone.js]: JS Docs missing.

File: modules/UI/videolayout/InteractiveMicrophone.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+        level = level < this.min ? this.min : level;
+        level = level > this.max ? this.max : level;
+
+        this.level = level;
+
+        return this;
+    }
+
+    redraw () {
```
Comment (yanas) [inline modules/UI/videolayout/InteractiveMicrophone.js]: JS Docs missing

File: modules/UI/videolayout/InteractiveMicrophone.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
+/* global $ */
+import UIUtil from "../util/UIUtil";
+
+export default class InteractiveMicrophone {
```
Comment (yanas) [inline modules/UI/videolayout/InteractiveMicrophone.js]: JS Docs missing.

**Comment by yanas** [comment]
I left some comments mainly about js docs. Can we also change the filler color to something more flashy and visible? Some kind of blue that we already use in the variables.

**Comment by yanas** [comment]
Jenkins test this please.

**Comment by m-voloshin** [comment]
Jenkins test this please.

**Comment by m-voloshin** [comment]
Jenkins test this please.

**Comment by m-voloshin** [comment]
Jenkins test this please.

File: modules/UI/videolayout/VideoLayout.js (hunk 1 above)
Comment (m-voloshin) [inline modules/UI/videolayout/VideoLayout.js]: I've found identical method that was already introduced.

**Comment by m-voloshin** [comment]
Alternative implementation with dots was provided.

**Pull request closed without merging**